### PR TITLE
Fixed residual counts to create plot and added wider x-axis for fhit …

### PR DESCRIPTION
**Pull request by igherzog**
I changed the spacing for the x-axis labels for easier reading and presenting along with removing the dtype=int conflict caused with the python 3 environment

### CODE REVIEW

File: hawc_hal/HAL.py [hunk 1]
```diff
@@ -410,9 +410,9 @@ def display_spectrum(self):
         return self._plot_spectrum(net_counts, yerr, model_only, residuals, residuals_err)
 
     def _plot_spectrum(self, net_counts, yerr, model_only, residuals, residuals_err):
-
+        plt.rcParams["figure.figsize"] = [12, 6]
         fig, subs = plt.subplots(2, 1, gridspec_kw={'height_ratios': [2, 1], 'hspace': 0})
```
Comment (henrikef) [inline hawc_hal/HAL.py]: Changing `plt.rcParams` will affect the defaults for all future plots in the same python session, so generally that's not something we'd want to change in a function like this. However, you can give the figure size as an argument to the `subplots` command, e.g. `fig, subs = plt.subplots( ..., figsize=(12,6) )`. That way, only the one figure will be affected.

**Review by henrikef — CHANGES_REQUESTED** [review]
Hi Ian! Thanks for this fix, please see my comment in the code.

Also, I assume you made some plots to make sure the new options work, could you upload an example here?

File: hawc_hal/HAL.py (hunk 1 above)
Comment (igherzog) [reply hawc_hal/HAL.py]: Sure, I'll make that change now (it returns the same plot format).  In the meantime, these are the 2 outcomes depending on whether fhit or gp is used.

hal_crab_residuals_gp
hal_crab_residuals

**Review by henrikef — APPROVED** [review]
Looks good!

**Pull request merged**